modules/nw-ptp-e810-hardware-configuration-reference.adoc

```diff
@@ -31,27 +31,39 @@ The `SMA2` connector is bidirectional.
 ====
 
 Set `spec.profile.plugins.e810.ublxCmds` parameters to configure the GNSS clock in the `PtpConfig` custom resource (CR).
+
+[IMPORTANT]
+====
+You must configure an offset value to compensate for T-GM GPS antenna cable signal delay.
+To configure the optimal T-GM antenna offset value, make precise measurements of the GNSS antenna cable signal delay.
+Red Hat cannot assist in this measurement or provide any values for the required delay offsets.
+====
+
 Each of these `ublxCmds` stanzas correspond to a configuration that is applied to the host NIC by using `ubxtool` commands.
 For example:
 
 [source,yaml]
 ----
 ublxCmds:
-  - args: #ubxtool -P 29.20 -z CFG-HW-ANT_CFG_VOLTCTRL,1
+  - args:
       - "-P"
       - "29.20"
       - "-z"
       - "CFG-HW-ANT_CFG_VOLTCTRL,1"
+      - "-z"
+      - "CFG-TP-ANT_CABLEDELAY,<antenna_delay_offset>"<1>
     reportOutput: false
 ----
+<1> Measured T-GM antenna delay offset in nanoseconds.
+To get the required delay offset value, you must measure the cable delay using external test equipment.
```

Comment:
```suggestion
<1> Measured T-GM antenna delay offset in nanoseconds. To get the required delay offset value, you must measure the cable delay using external test equipment.
```

- To avoid hard wrapping as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines

Reply:
One sentence per line is not "hard wrapping". See https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line